CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project aspires to adhere to [Semantic Versioning](https://semver.org/s
 #### General
 - Avoid compile issue with using `_Pragma()` with Python 3.8 on Windows
 
+#### Blueprint
+- Added the `blueprint::mesh::examples::polychain` example. It is an example of a polyhedral mesh. Also updated the docs to reflect this change.
```

Comment:
Small suggestion - Change: 

`Also updated the docs ....`

to:


See Mesh Blueprint Examples docs (https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh.html#polychain) for more details.